_includes/manuals/2.0/1.2_release_notes.md

```diff
@@ -442,6 +442,7 @@ External authentication sources can now be assigned to taxonomies using a new UI
 
 ### Upgrade warnings
 
+* The CentOS packages are not tested on Scientific Linux or Oracle Linux. The Foreman installation on Scientific Linux or Oracle Linux may or may not work.
```

Comment:
@ehelms IMHO we don't really have CentOS packages, but rather EL packages (that we build on CentOS). Do you have thoughts how we should phrase this? Something like:
```suggestion
* The RPM packages are not tested on Scientific Linux or Oracle Linux. The Foreman installation on Scientific Linux or Oracle Linux may or may not work.
```

Reply:
:+1: I think that wording sounds good

Reply:
Added suggested change, request to review. Thank you !